src/main.rs

```diff
@@ -413,7 +413,7 @@ fn main() {
 		config.channel_options.announced_channel = ANNOUNCE_CHANNELS;
 
 		let channel_manager = if let Ok(mut f) = fs::File::open(data_path.clone() + "/manager_data") {
-			let (last_block_hash, manager) = {
+			let (_last_block_hash, manager) = {
```

Comment:
Hmm, can you add a TODO like the one at the same line above here, to make sure its at least noted that there is reason rust is complaining that this is unused.

Reply:
There was one already, but I changed it to be equal to the one above.